it/controllers/benefits/NonQualifyingRelocationBenefitsControllerISpec.scala

```diff
@@ -0,0 +1,408 @@
+          lazy val result: WSResponse = {
+            dropEmploymentDB()
+            insertCyaData(employmentUserData(isPrior = true, cyaModel("employerName", hmrc = true,
+              Some(benefits(fullAccommodationRelocationModel.copy(nonQualifyingRelocationExpensesQuestion = None))))), userRequest)
+            authoriseAgentOrIndividual(user.isAgent)
+            urlGet(url(taxYearEOY), welsh = user.isWelsh, headers = Seq(HeaderNames.COOKIE -> playSessionCookies(taxYearEOY)))
+          }
+          "has an OK status" in {
+            result.status shouldBe OK
+          }
+
+          implicit def document: () => Document = () => Jsoup.parse(result.body)
+
+          titleCheck(user.specificExpectedResults.get.expectedTitle)
+          h1Check(user.specificExpectedResults.get.expectedH1)
+          captionCheck(expectedCaption)
+          textOnPageCheck(get.expectedContent, contentSelector)
+          textOnPageCheck(get.expectedExample1, contentExample1Selector)
+          textOnPageCheck(get.expectedExample2, contentExample2Selector)
+          buttonCheck(continueButtonText, continueButtonSelector)
+          formPostLinkCheck(continueLink, formSelector)
+          welshToggleCheck(user.isWelsh)
+
+          radioButtonCheck(yesText, 1, Some(false))
+          radioButtonCheck(noText, 2, Some(false))
+        }
+
+        "render the non-qualifying relocation benefits question page with cya data and 'yes' radio selected" which {
+          lazy val result: WSResponse = {
+            dropEmploymentDB()
+            insertCyaData(employmentUserData(isPrior = true, cyaModel("employerName", hmrc = true,
+              Some(benefits(fullAccommodationRelocationModel.copy(nonQualifyingRelocationExpensesQuestion = Some(true)))))), userRequest)
+            authoriseAgentOrIndividual(user.isAgent)
+            urlGet(url(taxYearEOY), welsh = user.isWelsh, headers = Seq(HeaderNames.COOKIE -> playSessionCookies(taxYearEOY)))
+          }
+          "has an OK status" in {
+            result.status shouldBe OK
+          }
+
+          implicit def document: () => Document = () => Jsoup.parse(result.body)
+
+          titleCheck(user.specificExpectedResults.get.expectedTitle)
+          h1Check(user.specificExpectedResults.get.expectedH1)
+          captionCheck(expectedCaption)
+          textOnPageCheck(get.expectedContent, contentSelector)
+          textOnPageCheck(get.expectedExample1, contentExample1Selector)
+          textOnPageCheck(get.expectedExample2, contentExample2Selector)
+          buttonCheck(continueButtonText, continueButtonSelector)
+          formPostLinkCheck(continueLink, formSelector)
+          welshToggleCheck(user.isWelsh)
+
+          radioButtonCheck(yesText, 1, Some(true))
+          radioButtonCheck(noText, 2, Some(false))
+        }
+
+        "redirect user to tax overview page if it's not EOY" which {
```

Comment:
This EOY redirect test could maybe be outside of the     userScenarios.foreach { user =>
as it is not a content check